src/chrome/content/rules/Axios.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Nonfunctional hosts in *.axios.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Axios.com">
+	<target host="axios.com" />
+	<target host="www.axios.com" />
+
```

Comment:
You may also want to add `<securecookie host=".+" name=".+" />` to these rules

Reply:
Thanks for the suggestion. I've added a supercookie line to each of the rules.

I realized after adding a few commits that I had misread the output of the CI test, but I still don't really understand what's failing. Is this a common thing? Do you happen to know what's wrong?

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/289194360 

```
ERROR src/chrome/content/rules/Infobae.com.xml: Fetch error: http://infobae.com/ => https://infobae.com/: (28, 'Connection timed out after 20000 milliseconds')
```